etc/libreoffice.profile

```diff
@@ -27,13 +27,13 @@ apparmor
 caps.drop all
 machine-id
 netfilter
-nodbus
+#nodbus
 nodvd
 nogroups
-nonewprivs
+#nonewprivs
 noroot
 notv
-protocol unix,inet,inet6
+#protocol unix,inet,inet6
```

Comment:
Why does this line need commenting?

Reply:
i got crazy errors at libreoffice

Reply:
@bn0785ac We still try to find a different solution to the problem with Libreoffice in #1703. `nodbus` (for KDE in general) and `apparmor` (for Kubuntu only) probably needs to be disabled, yes.

Reply:
What breaks apparmor?

Reply:
REAL errors :dagger:

Reply:
@Vincent43 D-Bus. Means issues are probably limited to Kubuntu, see #1703.

Reply:
no,  it happens here (manjaro *unstable* kde)


manjaro unstable = arch but with a 24 hours delay for update

Reply:
Can you paste the errors you get and describe what you see?
Thanks!